routers/api/v1/api.go

```diff
@@ -189,84 +189,119 @@ func reqBasicAuth() macaron.Handler {
 	}
 }
 
+func isSiteAdmin(ctx *context.Context) bool {
```

Comment:
Could these methods be added to API context?

Reply:
Good idea. I've moved it.